spec/intl.html

```diff
@@ -495,7 +511,9 @@ <h1>ToLocalTime ( _t_, _calendar_, _timeZone_ )</h1>
       </p>
 
       <emu-alg>
-        1. Assert: Type(_t_) is Number.
+        1. Assert: Type(_t_) is <del>Number</del><ins>BigInt</ins>.
+        1. <ins>Set _t_ to 𝔽(floor(ℝ(_t_) / 10<sup>6</sup>)).</ins>
+        1. <mark>TODO: This probably causes issues with fractional seconds. Review.</mark>
```

Comment:
Would we have to patch LocalTZA to accept epoch nanoseconds as well?

Reply:
Fixed by not using LocalTZA.